atom/browser/api/atom_api_desktop_capturer.cc

```diff
@@ -169,8 +169,14 @@ void DesktopCapturer::UpdateSourcesList(DesktopMediaList* list) {
       std::vector<std::string> device_names;
       // Crucially, this list of device names will be in the same order as
       // |media_list_sources|.
-      webrtc::DxgiDuplicatorController::Instance()->GetDeviceNames(
-          &device_names);
+      bool success =
+          webrtc::DxgiDuplicatorController::Instance()->GetDeviceNames(
```

Comment:
According to the api definition, it should return false only when DirectX capturing is not supported, in that case `using_directx_capturer_` should have never been true, weird.